### DISCUSSION TITLE
Forge crashing on startup

### DISCUSSION THREAD

**Discussion by natcazador** [microsoft/openjdk#544] [Q&A] [answered]
Installed current version of Java, as well as Minecraft. Used version 47.0.35 of forge.  Keeps crashing every time I attempt to launch. Help please.

Crash log

`22:40:04.333
game
#
22:40:04.333
game
# JRE version: OpenJDK Runtime Environment Microsoft-32931 (17.0.3+7) (build 17.0.3+7-LTS)
22:40:04.333
game
# Java VM: OpenJDK 64-Bit Server VM Microsoft-32931 (17.0.3+7-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
22:40:04.333
game
# Problematic frame:
22:40:04.375
game
# C  [atio6axx.dll+0x196280]
22:40:04.375
game
#
22:40:04.375
game
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
22:40:04.375
game
#
22:40:04.375
game
# An error report file with more information is saved as:
22:40:04.375
game
# C:\Users\turn_\AppData\Roaming\.minecraft\hs_err_pid13324.log
22:40:05.121
game
#
22:40:05.121
game
# If you would like to submit a bug report, please visit:
22:40:05.121
game
#   https://github.com/microsoft/openjdk/issues
22:40:05.121
game
# The crash happened outside the Java Virtual Machine in native code.
22:40:05.121
game
# See problematic frame for where to report the bug.
22:40:05.121
game
#
22:40:05.131
game
22:40:05.131
game
[error occurred during error reporting (), id 0xc0000005, EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ff90b25afc9]
22:40:05.131
game
22:40:05.175
monitor
Process Monitor
Process crashed with exit code 1`

**Comment by karianna** [comment, marked as answer]
Thank you for the report!
It looks like you are reporting that the game Minecraft crashed for you. Please try the following:
- Update your graphics drivers, see [this Minecraft Help Center article](https://help.minecraft.net/hc/en-us/articles/4409137348877-Minecraft-Java-Edition-Game-Crashes-and-Performance-Issues-FAQ), section "Update your Video Card Drivers";
  often Minecraft crashes are caused by outdated graphics drivers
- In case you are using mods, try disabling them one by one until you find the mod causing the crash, then report the crash to the mod author
- If you are using a third-party launcher, try the [official launcher](https://www.minecraft.net/download) instead

If none of the above helped, and you are still experiencing the crash with the official launcher and without any mods installed, please submit this issue over at the [Mojang bug tracker](https://bugs.mojang.com/projects/MC/summary). Please search for existing reports first; in case you do not find any, create a new report and let us know about the issue number here (e.g. `MC-123456`).
The Mojang team will take the first look. If an OpenJDK bug is identified, the Mojang team will contact the Microsoft Build of OpenJDK team to address the issue.
